common/utils/search.ts

```diff
@@ -121,3 +124,38 @@ export const hasFilters = ({
 }): boolean => {
   return !!filters.filter(element => queryParams.includes(element)).length;
 };
+
+// ROUTING
+/**
+ * Cleans up the URL for form submissions in order to avoid unwanted query parameters
```

Comment:
Could you provide an example of what sort of cleaning it does? A before/after of the URL.

Reply:
This sent me down a rabbit hole with @gestchild and we're now investigating if it's still needed at all.
I think this is a different PR though, so I've added a TODO and there is a thread to investigate https://wellcome.slack.com/archives/C3TQSF63C/p1677248039025289.